commands/build.go

```diff
@@ -486,7 +486,7 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions, debugConfig *debug.D
 
 	flags := cmd.Flags()
 
-	flags.StringSliceVar(&options.extraHosts, "add-host", []string{}, `Add a custom host-to-IP mapping (format: "host:ip")`)
+	flags.StringSliceVar(&options.extraHosts, "add-host", []string{}, `Add a custom host-to-IP mapping (format: "host=ip")`)
```

Comment:
We could keep the example as `:` here, until we officially deprecate the version. That would give users time to upgrade to later versions which would support both syntaxes.

Reply:
Ok, will do ...

@crazy-max - this will undo the earlier change for your https://github.com/docker/buildx/pull/2121#discussion_r1395737201 - but I think it makes sense given https://github.com/docker/buildx/pull/2121#discussion_r1401384032 ?

Reply:
Yes agree let's keep `:` for now